frameworks/keyed/ember/app/templates/components/bs-button.hbs

```diff
@@ -1,7 +1,7 @@
 <button
   type="button"
-  class="btn btn-primary btn-block" id={{@id}}
-  onclick={{action @onClick}}>
+  class="btn btn-primary btn-block"
+  onclick={{action @onClick}} ...attributes>
```

Comment:
I don't think this `action` is needed. could speed up some things if we aren't doing an extra bind

Reply:
it's static rendering, don't think it has an impact